reference/conanfile/methods/config_options.rst

```diff
@@ -0,0 +1,16 @@
+.. _reference_conanfile_methods_config_options:
+
+config_options()
```

Comment:
We can add a link to the tutorial: https://docs.conan.io/en/2.0/tutorial/creating_packages/configure_options_settings.html